lib/screens/explore_screens/regions_screens/region_event_details/region_event_details_screen.dart

```diff
@@ -57,6 +57,7 @@ class RegionEventDetailsScreen extends StatelessWidget {
             case PageState.loading:
               return const Scaffold(body: FullPageLoadingIndicator());
```

Comment:
Is this Loading really need it? we get the event from the navigation arguments.

Reply:
I think you use it in the delete of the event.

![image](https://user-images.githubusercontent.com/42857405/166588594-3ae84032-d1f7-435a-b10a-7a012d9a5d12.png)